src/goto-symex/path_storage.h

```diff
@@ -115,8 +116,24 @@ class path_storaget
   virtual patht &private_peek() = 0;
   virtual void private_pop() = 0;
 
+  typedef std::unordered_map<irep_idt, std::size_t> name_index_mapt;
+
+  std::size_t get_unique_index(
+    name_index_mapt &unique_index_map,
+    const irep_idt &id,
+    std::size_t minimum_index)
+  {
```

Comment:
definition should go to the cpp file. Also I would suggest declaring a class for `name_index_mapt` (instead of a typedef) and make this the only public method.

Reply:
Seems short and hot enough to be worth inlining to me